### PR TITLE
DK-441 Change end screen animation is implemented

### PR DESCRIPTION
**Pull request by pavelpavelko**

### CODE REVIEW

**Comment by alexbejann** [comment]
LGMT :D

**Review by alexbejann — APPROVED** [review]

File: lib/screens/qr_code_scan_screen.dart [hunk 1]
```diff
@@ -41,6 +41,9 @@ class _QrCodeScanScreenState extends State<QrCodeScanScreen> {
         (_) {
           Navigator.of(context).pushNamed(
             ChooseAmountSliderScreen.routeName,
+            arguments: Organisation(
+              name: "Mock Organisation",
+            ),
```
Comment (MaikelStuivenberg) [inline lib/screens/qr_code_scan_screen.dart:46]: Why do we need a mock name in here?

Comment (pavelpavelko) [reply lib/screens/qr_code_scan_screen.dart:46]: in this code we simulate the qr code scanning for the ios simulator, bc there is no camera emulation for the ios simulators. But we need the organisation name in the following screen, so. 
please take a look on the condition above:
`//temporary code to test on iOS simulators
    if (kDebugMode && Platform.isIOS) {`

Comment (MaikelStuivenberg) [reply lib/screens/qr_code_scan_screen.dart:46]: Ah okay! I was only at the changes :-)

**Review by MaikelStuivenberg — APPROVED** [review]

**Pull request merged**